modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +57,301 @@ docker compose version
 
 You should see the version of Docker Compose that's installed on your local machine.
 
-== Deploy Redpanda
+.Windows users:
```

Comment:
```suggestion
.Windows users
```
Can we replace "Note" with "Windows users"?

Reply:
I think so. I'll test it out and commit the change when it's working.

Reply:
OK done. The syntax is:

```asciidoc
:note-caption: Windows users

[NOTE]
====
If you're on Windows, use a compatible terminal that supports Unix-like commands. Or, consider using https://docs.microsoft.com/en-us/windows/wsl/install[Windows Subsystem for Linux (WSL)].
====

:note-caption: Note
```

The `:note-caption: Note` resets the title of the admonition back to 'NOTE' for future use in this doc.